### DISCUSSION TITLE
Provide “official” documentation of maven best practices

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#3] [Ideas for Funding]
With the introduction of Maven 4 the documentation of Maven is very outdated. We should target the most common use cases and templates and provide solid and open documentation for them. This documentation should be used by people that want to use Maven in a Java project.

**Comment by khmarbaise** [comment]
Not only "best practices" also a general documentation would be a good thing.
Some kind like that: https://the-maven-practical-guide.io/book/the-maven-practical-guide.html (Starting point)

**Reply by ascheman** [reply]
> Not only "best practices" also a general documentation would be a good thing. Some kind like that: https://the-maven-practical-guide.io/book/the-maven-practical-guide.html (Starting point)

Found [this repository](https://github.com/khmarbaise/the-maven-practical-guide) for this. 
Would you mind if someone contributes to it, @khmarbaise?

**Comment by ascheman** [comment]
Documentation in general could follow the [Diataxis approach](https://diataxis.fr/).
Also affects #2, #4, #5, #9, #10, #12, ...

**Comment by ascheman** [comment]
This discussion has been converted to Epic #55 for tracking.

Closing this discussion.
